src/store/index.ts

```diff
@@ -1,25 +1,26 @@
 import { defineStore } from 'pinia';
 import { getAllMoviesData, getMovieGenresData, getSeancesData } from '@/services/api/data.js';
+import { GenreData, MovieData, SeanceData, ScreeningData } from '@/types/data';
 import router from '@/router';
 
 export const useMainStore = defineStore('main', {
   state() {
     return {
-      error: null,
-      errorDisplayTime: 4,
+      error: null as Error,
```

Comment:
```suggestion
      error: null as Error | null,
```

Reply:
Yeah, after adding `"strict": true` to the config file, lDE complains about it as well, so it really helps a lot!